docs/architecture/modernize-desktop/example-migration-to-net-core.md

```diff
@@ -0,0 +1,365 @@
+in your code that can potentially cause problems at run-time with .NET Core. You
+can go to <http://github.com/dotnet/platform-compat> and use the Roselyn
+analyzers.
+
+## Migrating a Windows Forms application
+
+To showcase a complete migration process of a Windows Forms application, we've
+chosen to migrate the eShop sample application.
+
+This application shows a product catalog and allows the user to navigate, filter, and search for products. From an architecture point of view, the App relies on an
+external WCF service that serves as a façade to a back-end database.
+
+You can see the main application window in the following picture:
+
+![Main application window ](media/example-migration-to-net-core/main-application-window.png)
+
+If we open the .csproj file we can something like this:
+
+![.csproj file ](media/example-migration-to-net-core/csproj-file.png)
+
+As mentioned before, .NET Core project has a more compact style and we need to
+migrate the project structure to the new .NET Core SDK style.
+
+Select the Windows Forms project in the solution explorer and do *right click
+-> Unload Project -> Edit*
+
+![Unload Project](media/example-migration-to-net-core/unload-project.png)
+
+Now we can update the .csproj file. We will delete all the content and replace
+it with:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>netcoreapp3.0</TargetFramework>
+        <UseWindowsForms>true</UseWindowsForms>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    </PropertyGroup>
+</Project>
+```
+
+If we save and reload the project, we are done updating the project file. Now
+the project is targeting the .NET Core.
+
+If we compile the project at this point, we find some errors related to the WCF
+client reference. Since this is autogenerated code, we must regenerate it to
+target .NET Core.
+
+![Errors](media/example-migration-to-net-core/errors.png)
+
+We can delete the `Reference.cs` file and generate a new Service Client.
+
+Right-click over *Connected Services* and select the "*Add Connected Service*"
+option.
+
+![Add Connected Service](media/example-migration-to-net-core/add-connected-service.png)
+
+This will open the Connected Services window where we select the Microsoft WCF
+Web Service option.
+
+![Connected Services window](media/example-migration-to-net-core/connected-services-window.png)
+
+If we have the WCF Service in the same solution as we have, we can choose to
+select the Discover option instead of specifying a service URL.
+
+![Configure WCF Web Service Reference window](media/example-migration-to-net-core/configure-wcf-reference.png)
+
+Once the service is located, the tool reflects the API contract implemented by
+the service. We change the name of the Namespace to be eShopServiceReference:
+
+![API contract](media/example-migration-to-net-core/api-contract.png)
+
```

Comment:
The previous image would probably benefit of having the selection highlighted (others on the book would probably too)
Similar to this: https://docs.microsoft.com/en-us/dotnet/standard/analyzers/portability-analyzer#analyze-portability
Guidelines: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-create-screenshot?branch=master#add-red-boxes